internal/grill/runner.go

```diff
@@ -12,6 +13,9 @@ import (
 	"syscall"
 )
 
+// Token used to separate output of individual commands in a suite.
+const testBreak = "__GRILL__"
```

Comment:
Seems fragile, shouldn't it work by keeping track of indices?

Reply:
I can't think of how it would work because each test command may output more or less lines than what's expected. Cram uses similar line splits but with the addition of random salt, which I'm guessing is intended to minimize the chance of the separator occurring unintentionally in the test spec. I can add the same here.

Reply:
OK, sounds like a good compromise.

Reply:
Done; PTAL